lib/commands/waitForExist.js

```diff
@@ -65,7 +65,9 @@ module.exports = function waitForExist(selector, ms, reverse) {
 
                 var now = new Date().getTime();
                 if(now - start > ms) {
-                    return cb(null, false);
```

Comment:
If you wanted an assertion, I would assume you would want to assert whether or not the command succeeded
```javascript
var assert = require('chai').assert;
client.waitForExist('.textbox', function (error, result) {
    assert.notOk(error)
})
```
just like you would do on any other command that you expect might fail.   

The original issue #419 is that waitForExist doesn't behave like the other waitForX commands because it doesn't break the chain if the element is not found
```javascript
client.waitForExist('.textbox')
      .click('.textbox')
```
To quote you on this issue: 
> You're right, it's not completely clear in our docs. According to similar commands in other Selenium libraries and other commands in this project, all waitFor commands should break the chain if no callback is given and return value would be false. I will mark this as enhancement and will tackle it once I got time. Any PRs are welcome of course.

.

> all waitFor commands should break the chain if no callback is given and return value would be false

And I agree with the idea of not having to provide a custom callback for all waitForX commands that might fail.